framework/src/main/java/org/tron/core/db/PendingManager.java

```diff
@@ -22,6 +24,8 @@ public PendingManager(Manager db) {
         tmpTransactions.add(transactionCapsule);
       }
     });
+    MetricsUtil.meterMark(MetricsKey.BLOCKCHAIN_MISS_TRANSACTION,
```

Comment:
Logic error, missed should be a timeout transaction

Now the statistics are no overtime transactions

Reply:
ok.  will use  db.getPendingTransactions().size() - tmpTransactions.size() to get timeout Tx